docs/style-guide.md

```diff
@@ -221,3 +231,47 @@ For an example of when *not* to use lexicographic order, consider an array holdi
 Spec changes that add properties should do so in accordance with this recommendation, rather than automatically placing them at the end—relative enumeration order of properties should remain stable over time, but there is no such expectation regarding adjacency.
 #### Examples
 `Object.keys(new Intl.Segmenter().resolvedOptions())` returns `[ 'locale', 'granularity' ]`. If support for the [Unicode BCP 47 U Extension "dx" key](https://unicode.org/reports/tr35/#UnicodeDictionaryBreakExclusionIdentifier) were added and exposed as `dictionaryBreakExcludedScripts`, that property would belong before `granularity`.
+
+## Naming Conventions
+
+This section provides guidelines for naming conventions in ECMA-402, focusing on clarity, consistency, and adherence to established practices.
+
+### Avoiding Abbreviations in Property Names and Values
+
+:star2: *Property names and values in ECMA-402 should avoid abbreviations to ensure clarity and readability. Use descriptive names that clearly convey their purpose.* :star2:
+
+#### Guidelines
+
+- Use full, descriptive names for properties and values, even if they are longer.
+- Avoid cryptic abbreviations or shortened forms that may confuse readers or developers unfamiliar with the context.
+- Prioritize clarity over brevity, especially for properties that are frequently used or have significant impact.
+
+#### Rationale
+
+- Descriptive names improve code readability and maintainability.
+- They reduce the cognitive load for developers, especially those new to the specification.
```

Comment:
Also, spelled-out names are more accessible to non-native English speakers, because they are more googlable, etc. We are building an i18n library, after all.

Reply:
What about the immediately following point? I believe it comes close so at the very least I should add something along these lines to it (or after it).